docs/reference/api/storage/LittleFileSystem.md

```diff
@@ -12,181 +11,112 @@ for embedded systems, specifically microcontrollers using flash storage.
    | | |
 ```
 
-Microcontrollers and flash storage preset three big challenges for embedded
-storage: [power-loss](#power-loss-resilient), [wear](#wear-leveling), and
-[limited RAM and ROM](#bounded-ram-and-rom). The littlefs provides a solution
-to all three of these problems.
+Microcontrollers and flash storage present three challenges for embedded storage: [power loss](#power-loss-resilience), [wear](#wear-leveling) and [limited RAM and ROM](#bounded-ram-and-rom). The LittleFileSystem provides a solution to all three of these problems.
 
-### Power-loss resilient
+### Power loss resilience
 
-Embedded systems are usually designed without a shutdown routine and the notable
-lack of a user interface for recovery. With a filesystem that is not resilient
-to power-loss, you rely on luck to not end up with a corrupted filesystem.
-The combination of persistent storage and unpredictable power-loss creates
-difficult to notice bugs that will ruin the user-experience of unlucky users.
+Embedded systems are usually without a shutdown routine and the notable lack of a user interface for recovery. With a file system that is not resilient to power loss, you rely on luck to not end up with a corrupted file system. The combination of persistent storage and unpredictable power loss creates bugs that are difficult to notice and that ruin the experience of unlucky users.
```

Comment:
Hmmm
``` diff
- Embedded systems are usually without a shutdown routine and the notable lack of a user interface for recovery.
+ Embedded systems are usually without a shutdown routine and notably lack a user interface for recovery.
```

Reply:
Is this a bad idea?
``` diff
- bugs that are difficult to notice and that ruin the experience of unlucky users.
+ bugs that are difficult to notice and ruin the experience of unlucky users.
```

Reply:
"Embedded systems are usually without a shutdown routine and notably lack a user interface" - This claim sounds little controversial to me. If you consider the world on embedded systems, phones would be a biggest chunk(I think), and all of them have proper shutdown mechanism :-).If you say "IoT" embedded systems that claim would probably be true.

Reply:
Good point, we should add IoT to clarify.

I've also used the term "microcontroller-scale embedded systems", thoughts on this vs "IoT embedded systems"?